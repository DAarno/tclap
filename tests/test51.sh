#!/bin/sh
# failure
`dirname $0`/simple-test.sh `basename $0 .sh` test8  -s=one homer -B
