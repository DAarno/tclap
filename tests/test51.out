PARSE ERROR:  
             Required argument missing: unTest2

Brief USAGE: 
   ../examples/test8  [-Bh] [--version] [-f=float ...] [-i=int ...]
                      -s=string unTest1 unTest2 [fileName ...]

For complete USAGE and HELP type: 
   ../examples/test8 --help

